bin/checkCLA.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
```

Comment:
Do we need the CLA signed for this repo?

Reply:
No, we can remove this. Apache 2 is good, CLAs increase the level of entry for contributors

Reply:
:+1: